packages/adapter-node/package.json

```diff
@@ -2,12 +2,14 @@
 	"name": "@sveltejs/adapter-node",
 	"version": "1.0.0-next.11",
 	"main": "index.js",
+	"type": "module",
```

Comment:
we can't use `"type":"module"` here, because you can't import a module into `svelte.config.cjs` (and you can't use .js instead because that... breaks stuff)

Reply:
hmm sample apps are currently working and tests are passing with this. is there a special test path that I can take to see what is breaking with this?

Reply:
ahh i got it!

`node node svelte.config.cjs`


```
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/samccone/Desktop/repos/kit/packages/adapter-node/package.json.

    at new NodeError (node:internal/errors:329:5)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1109:13)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/Users/samccone/Desktop/repos/kit/examples/svelte-kit-demo/svelte.config.cjs:1:17)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32) {
  code: 'ERR_REQUIRE_ESM'
}```

I think there is a way to make this work with conditional exports actually from package.json ... lemme see if I can make it work!

Reply:
woot, got it working.

Reply:
oh, nice